packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -18,8 +18,10 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 - Added a new type `DataSetAllocTemplate` for data set attributes.
 - Added optional `cancelJob` function to `ZoweExplorerApi.IJes` interface.
 - Added z/OSMF API implementation for `cancelJob` function.
+- Added optional `profile` parameter to `IPromptCredentialsOptions` so developers can choose to skip rebuilding the profile with ProfilesCache.
```

Comment:
Hey @traeok can we move the changelog items under 2.9.1?  Are they on main under 2.9.0 but didn't go out with that release?  If so we can fix when creating PR from maintenance to main to bring in 2.9.1 changes.

Reply:
Yes, sorry for the confusion - will update. I think it was added to the changelog under 2.9 at the time it was marked as ready, but was merged after the release.

Reply:
no worries, thanks for updating.  The same situation slipped by me last time till it was pointed out in release PR (thank goodness 😆 )